src/e2e/java/uk/gov/hmcts/reform/sscs/functional/sya/SubmitAppealTest.java

```diff
@@ -124,7 +124,7 @@ private void assertCaseIsExpectedResult(String expectedBody, String expectedStat
 
         assertEquals(expectedState, sscsCaseDetails.getState());
 
-        assertEquals("Joe Bloggs", sscsCaseDetails.getData().getCaseName());
+        //assertEquals("Joe Bloggs", sscsCaseDetails.getData().getCaseName());
```

Comment:
Guess take this out before merging

Reply:
Yes. Confirmed with Chris that it should be removed but he might merge before this PR
Will remove it in my PR